tests/examples/source/devfiles/nodejs/devfile-with-volumes.yaml

```diff
@@ -1,53 +1,52 @@
-apiVersion: 1.0.0
+schemaVersion: 2.0.0
 metadata:
   name: test-devfile
 projects:
-  -
-    name: nodejs-web-app
-    source:
-      type: git
+  - name: nodejs-web-app
+    git:
       location: "https://github.com/che-samples/web-nodejs-sample.git"
 components:
-  - type: dockerimage
-    image: quay.io/eclipse/che-nodejs10-ubi:nightly
-    endpoints:
-      - name: "3000/tcp"
-        port: 3000
-    alias: runtime
-    env:
-      - name: FOO
-        value: "bar"
-    memoryLimit: 1024Mi
-    mountSources: true
-    volumes:
-      - name: myvol
-        containerPath: /data
-  - type: dockerimage
-    image: quay.io/eclipse/che-nodejs10-ubi:nightly
-    alias: runtime2
-    memoryLimit: 1024Mi
-    mountSources: false
-    volumes:
-      - name: myvol
-        containerPath: /data
-      - name: myvol2
-        containerPath: /data2
+  - container:
+      name: runtime
+      image: quay.io/eclipse/che-nodejs10-ubi:nightly
+      memoryLimit: 1024Mi
+      env:
+        - name: FOO
+          value: "bar"
+      endpoints:
+        - name: "3000/tcp"
+          configuration:
+            protocol: tcp
+            scheme: http
+          targetPort: 3000
+      mountSources: true
+      volumes:
+        - name: myvol
+          containerPath: /data
+  - container:
+      name: runtime2
+      image: quay.io/eclipse/che-nodejs10-ubi:nightly
+      memoryLimit: 1024Mi
+      mountSources: false
+      volumes:
```

Comment:
volume needs to specified like this
https://github.com/elsony/devfile2-registry/blob/master/devfiles/eclipse/springBoot/1.0.0/devfile.yaml#L20

Reply:
fixed